WordPress/Classes/ViewRelated/Reader/ReaderPostDetailViewController.h

```diff
@@ -1,9 +1,11 @@
 #import <UIKit/UIKit.h>
+#import "ReaderPostsViewController.h"
 
 @class ReaderPost;
 
 @interface ReaderPostDetailViewController : UIViewController
 
+@property (nonatomic, assign) ReaderViewStyle readerViewStyle;
```

Comment:
Annoying comment: (just for the sake of consistency) can we have this one as `readwrite` as well?